### PR TITLE
Create UI

### PR DESCRIPTION
**Pull request by Leboroz**
# Objectives

- Build interactive console apps.
- Run a program using the command line.

### CODE REVIEW

File: src/app.rb [hunk 1]
```diff
@@ -0,0 +1,138 @@
...
+  end
+
+  def self.create_person
+    puts 'Input name'
+    print 'Name: '
+    name = gets.chomp
+    puts 'Input age'
+    print 'Age: '
+    age = Integer(gets.chomp)
+    @@people.push(yield(age, name))
```
Comment (MahmoudBakr23) [inline src/app.rb]: - [ ] I've got the following error while I'm trying to create a new person, please double-check the implementation of your method

`uninitialized class variable @@people in App (NameError)`

File: src/app.rb [hunk 2]
```diff
@@ -0,0 +1,138 @@
...
+    puts 'Input title and author'
+    print 'Title: '
+    title = gets.chomp
+    puts ' '
+    print 'Author: '
+    author = gets.chomp
+    current = @@books.length
+    create_book(title, author)
+    puts current < @@books.length ? 'Book added' : 'Error'
+    @@books.push(Book.new(title, author))
```
Comment (MahmoudBakr23) [inline src/app.rb]: - [ ] I've got the following error while I'm trying to create a new book, please double-check the implementation of your method

`uninitialized class variable @@book in App (NameError)`

File: src/app.rb [hunk 3]
```diff
@@ -0,0 +1,138 @@
...
+3 - Create a person.
+4 - Create a book.
+5 - Create a rental.
+6 - List all rentals for a given person id.
+7 - Quit".freeze
+  def initialize
+    @books = []
+    @people = []
+    @rentals = []
+    @date = Time.new.strftime('%Y-%m-%d')
```
Comment (MahmoudBakr23) [inline src/app.rb:20]: **Hint**

- Please keep in mind that you specified the initials of your instance variables of the `App` class, and then you tried to call them as class variables with the double `@@`, and not single `@`

**Review by MahmoudBakr23 — CHANGES_REQUESTED** [review]
Hello @Leboroz 😃, 

Great work and effort you put into this project so far.

## Highlights 🚀 

There are some excellent points that you have made perfectly. Keep up! 💪
- Nice UI design

However, there is always room for improvement.

> Keep in mind that your long hours of work are appreciated. However, the code review process is for improving the code you wrote, and not for criticizing it. 😄

 ## Required Changes ♻️
_Please check the comments under the review._

## Optional suggestions
_Every comment with the [OPTIONAL] prefix is not crucial enough to stop the approval of this PR. However, I strongly recommend you to take them into account as they can make your code better._

Cheers and Happy coding!👏👨‍💻💻

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561) you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

**Review by BrianSammit — APPROVED** [review]
# STATUS: APPROVED :green_circle: :clap: 

Hello @Leboroz, 
Your project is complete!  There is nothing else to say other than... it's time to merge it :shipit: 
Congratulations! 🎉 

TO HIGHLIGHT:

- Your project is professional 
- Your repo is professional 
- Well structured files 


Now you can merge this branch and move on :1st_place_medal: 
HAPPY CODING AND GOOD LUCK WITH THE COMMING ONES :crossed_fingers: :four_leaf_clover: 

well-1

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
Please, remember to tag me in your question so I can receive the notification.

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561)  you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

**Pull request closed without merging**